scalalib/src/mill/javalib/revapi/RevapiModule.scala

```diff
@@ -0,0 +1,121 @@
+package mill.javalib.revapi
+
+import mill._
+import mill.javalib._
+import mill.javalib.revapi.RevapiModule.optional
+import mill.scalalib.publish.Artifact
+import mill.util.Jvm
+
+/**
+ * Adds support for [[https://revapi.org/revapi-site/main/index.html Revapi checker]] to enable API analysis and change tracking.
+ */
+@mill.api.experimental // until Revapi has a stable release
+trait RevapiModule extends PublishModule {
+
+  /**
+   * Runs [[revapiCliVersion Revapi CLI]] on this module's archives.
+   */
+  def revapi(args: String*): Command[Unit] = Task.Command {
+    val workingDir = millSourcePath
```

Comment:
It seems that this is only used for caching according to the upstream code linked below; maybe we can use `Task.dest` instead?

Reply:
Updated.
This should now allow for specifying relative paths for any extension [output](https://github.com/revapi/revapi/blob/69445626881347fbf7811a4a78ff230fe152a2dc/revapi-reporter-file-base/src/main/resources/org/revapi/reporter/file/default-schema.json#L16).